### PR TITLE
refactor(common): move LDML keyboard .xml reader/writer and kmx-plus builder to @keymanapp/developer-utils 🐉

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates-to: #9665

@keymanapp-test-bot skip

### CODE REVIEW

**Review by rc-swag — APPROVED** [review]
I can see at a high level location of files has moved, whether logically that works for the build I don't have insight.  Also changing the let to consts for the data type declarations. 
LGTM
Not sure if you want to wait for @srl295  approval also though

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.84-alpha](https://keyman.com/downloads/releases/alpha/18.0.84)

**Pull request merged**
